README.rst

```diff
@@ -138,15 +138,6 @@ dependencies with
 
 .. end-include-marker-installation-section
 
-.. start-include-marker-python-version-support-section
-
-Python version support
-----------------------
-
-Python 3.6, 3.7, and 3.8
-
-.. end-include-marker-python-version-support-section
-
```

Comment:
`README.rst` already has the PyPI version badges. Removing this little section also means we have one fewer spot to potentially forget to update moving forward.